crates/sui-core/src/authority.rs

```diff
@@ -3724,7 +3679,8 @@ impl AuthorityState {
     }
 
     pub async fn insert_genesis_object(&self, object: Object) {
-        self.database
+        // TODO(cache)
```

Comment:
What is this TODO for?

Reply:
it was to remove the `self.database`, which i did. removing TODO